stagemonitor-core/src/test/java/org/stagemonitor/configuration/source/ConfigServerConfigurationSourceTest.java

```diff
@@ -0,0 +1,236 @@
+package org.stagemonitor.configuration.source;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.BlockJUnit4ClassRunner;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+import org.stagemonitor.core.configuration.SpringCloudConfigConfigurationSource;
+import org.stagemonitor.core.util.HttpClient;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.util.Map;
+
+@RunWith(BlockJUnit4ClassRunner.class)
```

Comment:
What does the `BlockJUnit4ClassRunner` do in this case?

Reply:
Nothing, remainings of copy pasta. Removed.